package.json

```diff
@@ -71,16 +86,22 @@
     "uglify-js": "3.4.7"
   },
   "scripts": {
-    "build": "npm install && grunt",
+    "build": "npm install && npm run build-all-variants",
+    "build-all-variants": "grunt custom:slim --esm --filename=jquery.slim.mjs && grunt custom --esm --filename=jquery.mjs && grunt custom:slim --filename=jquery.slim.js && grunt",
     "start": "grunt watch",
-    "test:browserless": "grunt && grunt test:slow",
+    "test:browserless": "grunt && npm run test:node_smoke_tests && grunt test:slow",
     "test:browser": "grunt && grunt karma:main",
     "test:esmodules": "grunt && grunt karma:esmodules",
     "test:amd": "grunt && grunt karma:amd",
     "test:no-deprecated": "grunt test:prepare && grunt custom:-deprecated && grunt karma:main",
     "test:selector-native": "grunt test:prepare && grunt custom:-selector && grunt karma:main",
     "test:slim": "grunt test:prepare && grunt custom:slim && grunt karma:main",
-    "test": "npm run test:slim && npm run test:no-deprecated && npm run test:selector-native && grunt && grunt test:slow && grunt karma:main && grunt karma:esmodules && grunt karma:amd",
+    "test:node_smoke_tests:slim": "grunt node_smoke_tests:commonjs:./dist/jquery.slim.js && grunt node_smoke_tests:commonjs:jquery/slim",
```

Comment:
This first run node smoke tests by running `require( "FULL/PATH/TO/dist/jquery.slim.js" )` and then by running `require( "jquery/slim" )`; the latter part goes through `exports` defined in this `package.json`, which is a nice test that this setup works.

The same is done for other `test:node_smoke_tests:*` scripts.